### DISCUSSION TITLE
background settings not work

### DISCUSSION THREAD

**Discussion by rickywu** [ayamir/nvimdots#806] [Q&A] [answered]
settings["background"] = "light"

when startup and open nvimtree, the statusline is light background
![image](https://github.com/ayamir/nvimdots/assets/442398/f2bde52d-b69a-4664-9f6a-b7c5fd4a446e)

but changed to dark after open any file
![image](https://github.com/ayamir/nvimdots/assets/442398/1f17307c-8bf3-4b0f-82a9-bf0bfbf027d2)

**Comment by charliie-dev** [comment]
U need to restart nvim

**Comment by rickywu** [comment]
> U need to restart nvim

restarted many times

**Comment by ayamir** [comment]
What colorscheme do you use? Set background here won't change the theme of `lualine`.
Set it in this line:
https://github.com/ayamir/nvimdots/blob/a8e986eaf87801bb1be616ef9ffb18bcf39680ae/lua/modules/configs/ui/lualine.lua#L107

**Comment by rickywu** [comment]
thanks, solved

**Comment by rickywu** [comment]
lua\user\configs\lualine.lua:

return {
	options = { theme  = 'ayu_light', },
}

But got error, can you help
d:\DevTools\Neovim\nvim/lua/modules/utils/init.lua:258: bad argument #1 to 'pairs' (table expected, got string)

**Reply by ayamir** [reply]
create a folder named `ui` under `lua/user/configs/` and put `lualine.lua` to `lua/user/configs/ui/`.

**Reply by rickywu** [reply]
No error now but  not work
lua\user\configs\ui\lualine.lua

```
return {
	options = {
		theme  = 'ayu_light',
	},
}
```

**Reply by Jint-lzxy** [reply]
Does #1031 fix this?

**Reply by ayamir** [reply, marked as answer]
https://github.com/ayamir/nvimdots/pull/1039 will fixed it.
But I don't recommend changing the theme of `lualine` in this way.
Install the [theme](https://github.com/Shatur/neovim-ayu) first then change `colorscheme` and `background` in `user/settings.lua` instead.
```lua
settings["colorscheme"] = "ayu"

settings["background"] = "light"
```
